### PR TITLE
Only support mul-sum distributed matmul test for ampere and hopper

### PR DESCRIPTION
**Pull request by cowanmeg**

### CODE REVIEW

**Review by xwang233 — APPROVED** [review]

File: tests/cpp/test_multidevice_matmul.cpp [hunk 1]
```diff
@@ -62,6 +62,10 @@ TEST_F(DistributedMatmulTest, MulSum_LayoutTN_NoComms) {
   // MmaLayout::TN A(T), B(N), C(T)
   // A and C are sharded on dimension M
   // Tests local matmul with no communication
+  auto dev_prop = at::cuda::getCurrentDeviceProperties();
+  if (dev_prop->major > 9) {
+    GTEST_SKIP() << "Require Ampere or Hopper for generated matmul";
```
Comment (jacobhinkle) [inline tests/cpp/test_multidevice_matmul.cpp:67]: Note that you could use one of these macros if it helps: https://github.com/NVIDIA/Fuser/blob/f097e58ea1ae9d2c70115ce6dfabc7b27763fcb3/tests/cpp/utils.h#L510

**Comment by cowanmeg** [comment]
!build

File: tests/cpp/test_multidevice_matmul.cpp (hunk 1 above)
Comment (cowanmeg) [reply tests/cpp/test_multidevice_matmul.cpp:67]: Thanks!

**Comment by cowanmeg** [comment]
!build

**Pull request merged**